python/pyspark/ml/feature.py

```diff
@@ -1227,6 +1228,54 @@ def surrogateDF(self):
         return self._call_java("surrogateDF")
 
 
+@inherit_doc
+class Interaction(JavaTransformer, HasInputCols, HasOutputCol, JavaMLReadable, JavaMLWritable):
```

Comment:
This is relatively minor, but since this implements `JavaMLReadable` & `JavaMLWriteable` it might be reasonable to have a test for the read/write either in the doc tests or in `tests.py` -- it would not have to be very involved just taking model and roundtriping it write/read.

Reply:
That's a good point. I'd overlooked the fact that all the other classes in feature.py have a similar test. Will look into adding one.